x/crosschain/keeper/evm_deposit.go

```diff
@@ -36,11 +39,21 @@ func (k Keeper) HandleEVMDeposit(ctx sdk.Context, cctx *types.CrossChainTx) (boo
 	}
 
 	if inboundCoinType == coin.CoinType_Zeta {
-		// if coin type is Zeta, this is a deposit ZETA to zEVM cctx.
-		err := k.fungibleKeeper.DepositCoinZeta(ctx, to, inboundAmount)
+		indexBytes, err := cctx.GetCCTXIndexBytes()
 		if err != nil {
 			return false, err
 		}
+		data, err := base64.StdEncoding.DecodeString(cctx.RelayedMessage)
+		if err != nil {
+			return false, errors.Wrap(types.ErrUnableToDecodeMessageString, err.Error())
```

Comment:
Does it mean the cctx will be aborted?
I think we should revert if the message is not valid

Reply:
I feel like it's safer to abort; aborted tx can still be refunded. I have not thought it through, but reverting here allows an end user to create unnecessary outbounds . What do you think ?

Reply:
>but reverting here allows an end user to create unnecessary outbounds . What do you think ?
What do you mean? The user pay for the revert gas so this is fine .
Aborting CCTX should be by definition a unexcepted situation, something the user can't cause with their input otherwise this create a bad UX, 
This is what this task is about as well for example: https://github.com/zeta-chain/node/issues/2037 there is an aborted CCTX because the user set a invalid BTC address for an omnichain smart contract call,

Reply:
9f684b6de76eb43b1f5d3308ce165de390c535b0